webrtc.html

```diff
@@ -4062,7 +4220,9 @@ <h3 id="sec.identity-proxy-assertion-request">Requesting Identity
         <li>The <code>RTCPeerConnection</code> sends a "SIGN" message to the IdP
         proxy. This message includes the material
         the <code>RTCPeerConnection</code> desires to be bound to the user's
-        identity.</li>
+        identity.  This includes the fingerprint of the certificate that was
+        selected or generated during the construction of
```

Comment:
Given that "certificates" above is plural and "certificate" here is singular:
Do we have an interface for seeling which of the certificates from the "certificates" option is actually selected during the construction of the RTCPeerConnection?
Is this certificate selected during construction or during connection?